docs/system_overview/configuration.md

```diff
@@ -14,254 +14,676 @@ This scrambled password file is saved after an `iinit` is run. If this file does
+            "zone_port": 1247 
+        },
 
-This file contains the following top level entries:
+        // ... Additional Entries ...
+    ],
+
+    // See "Host Access Control" section for more details.
+    "host_access_control": {
+        // Defines zero or more objects used to control access to the local iRODS instance.
+        "access_entries": [
+            // Defines a single access constraint.
+            {
+                // Defines which user is allowed access to the server.
+                "user": "",
+
+                // Defines which group is allowed access to the server.
+                "group": "",
+
+                // With the "mask" property, defines which host(s) are allowed access to the server.
+                "address": "",
+
+                // With the "address" property, defines which host(s) are allowed access to the server.
+                "mask": ""
+            },
+
+            // ... Additional Entries ...
+        ]
+    },
+
+    // See "Host Resolution" section for more details.
+    "host_resolution": {
+        // Defines zero or more objects containing hostname resolution information for
+        // local or remote iRODS servers.
+        "host_entries": [
+            // Defines a single resolution rule.
+            {
+                // Identifies whether the hostnames defined under "addresses" represent local or remote
+                // hostnames.
+                //
+                // The following values are supported: local, remote
+                "address_type": "",
+
+                // A list of strings defining hostname aliases.
+                "addresses": []
+            },
+
+            // ... Additional Entries ...
+        ]
+    },
+
+    // Holds the log level for various log message categories used throughout the server.
+    // Missing log categories default to "info".
+    // 
+    // The following values are supported: trace, debug, info, warn, error, critical
+    "log_level": {
+        "agent": "info",
+        "agent_factory": "info",
+        "api": "info",
+        "authentication": "info",
+        "database": "info",
+        "delay_server": "info",
+        "legacy": "info",
+        "microservice": "info",
+        "network": "info",
+        "resource": "info",
+        "rule_engine": "info",
+        "server": "info"
+    },
+
+    // Indicates to iRODS whether to use the hash used by the client or the data at rest, or to force
+    // the use of the default hash scheme: strict or compatible.
+    "match_hash_policy": "compatible",
+
+    // A 32-byte encryption key shared by the zone for use in the advanced negotiation handshake at the
+    // beginning of an iRODS client connection.
+    "negotiation_key": "",
+
+    // Holds several different properties such as database connection information, rule engine configuration, etc.
+    "plugin_configuration": {
+        // TODO
+        "authentication": {},
```

Comment:
Needs a description.

When is this used? `izonereport` perhaps? Anything else?

Reply:
This doesn't appear to be used/set by anything.

We might be able to remove this.

Reply:
I've marked these as "This property can be ignored."